torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -728,6 +909,13 @@ struct CUDAFusionFunction : public CompiledFusionFunction {
     TORCH_CUDA_CHECK(cudaGetDeviceProperties(&prop, agraph.device));
     checkCUDAVersion(prop);
 
+    // Use a graph without the fused chunk nodes to emit a kernel.
```

Comment:
It doesn't seem right for implementation details like having fused chunks to leak into the CUDA/CPU backend code and get repeated twice. This will be hard to maintain if every new feature added to the fuser had to do something similar.